pkgs/http/test/io/client_test.dart

```diff
@@ -81,9 +80,8 @@ void main() {
       ..headers[HttpHeaders.userAgentHeader] = 'Dart';
 
     var responseFuture = client.send(request);
-    request
-      ..sink.add('{"hello": "world"}'.codeUnits)
-      ..sink.close();
+    request.sink.add('{"hello": "world"}'.codeUnits);
+    await request.sink.close();
```

Comment:
None of the `await`s in the tests are necessary, right? Otherwise this would be a breaking change.